components/ReportList.vue

```diff
@@ -97,15 +97,12 @@ const getClasses = (dep) => {
         </div>
       </RadioGroup>
     </div>
-    <button
-      type="button"
+    <business-button
       :disabled="isLoading"
-      :class="[
-        'inline-flex items-center border border-transparent py-4 text-lg font-medium text-slate-50 disabled:opacity-60 disabled:cursor-default sticky bottom-0 w-full justify-center bg-green-600 hover:bg-green-700',
```

Comment:
Should we make a sucess version?

Reply:
No, I think this should just be a one-off button. See the comments in https://github.com/axodotdev/turbo-axo/pull/161#issuecomment-1334102035 (& add your notes!)

For now, reverting this change should be ok.

Reply:
Reverted to just add the needed classes :)